front/app/components/admin/ImageCropper/index.tsx

```diff
@@ -67,7 +78,27 @@ const ImageCropper = ({
             onCropChange={handleCropChange}
             onCropComplete={onCropComplete}
             objectFit="contain"
+            onMediaLoaded={handleMediaLoaded}
           />
+
+          {/* 3:1 visual overlay inside crop area for mobile view */}
+          {mediaSize && show3x1MobileCropLines && (
+            <div
+              style={{
+                position: 'absolute',
+                top: '50%',
+                left: '50%',
+                width: `${mediaSize.width * 0.75}px`,
+                height: `110px`,
+                maxHeight: `${mediaSize.height}px`,
+                transform: 'translate(-50%, -50%)',
+                borderLeft: '2px dashed rgba(255, 255, 255, 0.8)',
+                borderRight: '2px dashed rgba(255, 255, 255, 0.8)',
+                pointerEvents: 'none',
+              }}
+            />
+          )}
```

Comment:
Wondering if I should try and generalize this more though? :thinking:

Reply:
Are we using this in another place besides this `ImageCropper`component? If not, I think it's totally fine like this

Reply:
Nope just here! Ok sounds good :+1: